docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -2,7 +2,17 @@
  "cells": [
```

Comment:
<p>In both of these, I think it's also helpful to point out the between scenario comparisons. It'll repeat the above info in the tables but this is the percent change in wasting with and without SQLNS which is cool. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='138'/>

Reply:
<p>Good idea!</p>